experiments/configurations/pgvector-single-node.json

```
@@ -39,7 +39,7 @@
         "upload_params": { "parallel": 16 }
     },
     {
-        "name": "qdrant-m-32-ef-128",
+        "name": "pgvector-m-32-ef-128",
         "engine": "qdrant",
```

We probably also need to change the engine here to be pgvector?

```suggestion
        "engine": "pgvector",
```

Also for the 4 other entries.